mlflow/utils/databricks_utils.py

```diff
@@ -303,18 +303,6 @@ def get_job_type():
         return _get_context_tag("jobTaskType")
 
 
-@_use_repl_context_if_available("jobType")
-def get_job_type_info():
-    try:
-        return _get_context_tag("jobType")
-    except Exception:
-        return None
```

Comment:
Let's keep this here just in case it's used by feature store or another team internally

Reply:
good call. Added back in :)